src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -16,23 +17,53 @@ const RadioButtonGroup = (props) => {
     Logger.deprecate('`styleOverride` that is used in the `RadioButtonGroup` component is deprecated and will soon be removed.');
   }
   const {
-    children, name, legend, error, warning, info, onBlur,
-    onChange, value, inline, legendInline, legendWidth, legendAlign,
-    legendSpacing, ml, mb, labelSpacing = 1, styleOverride
+    children,
+    name,
+    legend,
+    error,
+    warning,
+    info,
+    onBlur,
+    onChange,
+    value,
+    inline = false,
+    legendInline = false,
+    legendWidth,
+    legendAlign,
+    legendSpacing,
+    ml,
+    mb,
+    labelSpacing = 1,
+    adaptiveLegendBreakpoint,
+    adaptiveSpacingBreakpoint,
+    styleOverride = {}
   } = props;
 
+  const isAboveLegendBreakpoint = useIsAboveBreakpoint(adaptiveLegendBreakpoint);
+  const isAboveSpacingBreakpoint = useIsAboveBreakpoint(adaptiveSpacingBreakpoint);
+
+  let inlineLegend = legendInline;
+  if (adaptiveLegendBreakpoint && legendInline) {
+    inlineLegend = isAboveLegendBreakpoint;
+  }
+
+  let marginLeft = ml;
+  if (adaptiveSpacingBreakpoint && !isAboveSpacingBreakpoint) {
```

Comment:
If we make the hook return `false` we can simplify this expression
```suggestion
  if (!isAboveSpacingBreakpoint) {
```

Reply:
As above I don't think this would work